isis/tests/Statistics.cpp

```diff
@@ -0,0 +1,143 @@
+#include "Statistics.h" 
+
+#include "FileName.h"
+#include "IException.h"
+#include "Preference.h"
+#include "Statistics.h"
+#include "XmlStackedHandlerReader.h"
+
+#include <QDebug>
+#include <QDataStream>
+#include <QXmlStreamWriter>
+#include <QXmlInputSource>
+
+#include <float.h>
+
+#include <gtest/gtest.h>
+
+using namespace Isis;
+using namespace std;
+
+#define TEST_EPSILON 1e-8
+
+/**
+ * @author 2018-11-20 Tyler Wilson
+ *
+ * @internal
+ *   @history 2018-11-20 Tyler Wilson - Coverage of existing
+ *            statistical routines using truth data generated
+ *            in R.
+ */
+
+
+
+TEST(Statistics, HandCalculations) {
+
+    Statistics s;
+    
+    s.AddData(-10.0);
+    s.AddData(20.0);
+    s.AddData(30.0);
+    
+
+    //Values were computed in R
+    double mu =13.3333333333333;
+    double std = 20.8166599946613;
+    double var = 433.333333333333;
+    
+    double rms = 21.6024689946929;   
```

Comment:
There's only three data points. I think it would be better to just do the math right here.